python/ray/data/preprocessors/normalizer.py

```diff
@@ -7,16 +7,69 @@
 
 
 class Normalizer(Preprocessor):
-    """Normalize each record to have unit norm.
+    r"""Scales each sample to have unit norm.
 
-    Supports the following normalization types:
-        * l1: Sum of the absolute values.
-        * l2: Square root of the sum of the squared values.
-        * max: Maximum value.
+    This preprocessor works by dividing each sample (i.e., row) by the sample's norm.
+    The general formula is given by
+
+    .. math::
+
+        s' = \frac{s}{\lVert s \rVert_p}
+
+    where :math:`s` is the sample, :math:`s'` is the transformed sample, and :math:`p`
+    is the norm type.
+
+    The following norms are supporting:
```

Comment:
```suggestion
    The following norm types are supported:
```

Reply:
Fixed. Don't think the word "types" is necessary.`l1` are `l2` are definitionally norms.